plugins/autocertkit

```diff
@@ -1203,6 +1203,18 @@ def remove_route(session, args):
     return json_dumps("OK")
 
 
+@log_exceptions
+def set_kernel_parameter(key, value, mip=None):
+    """
+    Delete all line which contains key in config file, and then write key=value to file,
+    Will take effect after reload(sysctl --sytstem) or reboot
+    """
+    conf_file = "/etc/sysctl.d/ack.conf"
+    cmd = "sed -i '/^%s=/d' %s; echo '%s=%s'>>%s" % (key, conf_file, key, value, conf_file)
```

Comment:
You could also rewrite the right hand side of the assignment if you can be sure such an entry exists.
```
sed -i.bak  '/^key *=/s/=.*$/=value/' file
```

Reply:
Understood, thanks!